src/references/damageable_objects.md

```diff
@@ -0,0 +1,93 @@
+---
+id: damageable_reference
+name: Damageable Objects Reference
+title: Damageable Objects Reference
+tags:
+    - Reference
+---
+
+# Damageable Objects
+
+## Summary
+
+The **Damageable Object** is a **Core Object** that implements the **Damageable** interface and has properties, events, and functions for objects which can take damage and die.
+
+The **Damageable Object** makes tracking damage and destroying objects very easy. Children of a **Damageable Object** that are hit by a projectile (or another damage source), will look for an ancestor that is a **Damageable** and automatically deduct the damage from health of the **Damageable**. You have control over what happens when the health of the **Damageable Object** reaches 0.
+
+A benefit to **Damageable Objects**, is no scripting is needed to damage an object and destroy it. This can all be controlled from the custom properties, which makes it quick and easy to setup.
+
+## Use Cases
+
+There are many use cases for the **Damageable Object**. Here are a few examples of how the **Damageable Object** could be used in a game.
+
+- Breakable Objects
+
+    Some games have objects that can be broken and reveal loot for the player. For example, a Crate that can be destroyed by players. This can be done without any scripts.
```

Comment:
Instead of a single example, go more abstract. Breakable containers, like crates and pots ...

You could also reference specifically what event would be used to spawn the loot.

Again, all of this is to emphasize that this will not actually give creators a crate that brakes. It will give you them Lua to break the crate themselves.